core/cautils/normalize_image_name.go

```diff
@@ -1,10 +1,10 @@
-package opaprocessor
+package cautils
 
 import (
 	"github.com/docker/distribution/reference"
 )
 
-func normalize_image_name(img string) (string, error) {
+func NormalizeImageName(img string) (string, error) {
```

Comment:
@YiscahLevySilas1 I missed your reply, does this affect the latest rego changes?

Reply:
the rego uses [image.parse_normalized_name](https://github.com/kubescape/regolibrary/blob/623bec0346a1caed310a7c3832f5f1640ca0f085/rules/container-image-repository-v1/raw.rego#L8).

afaik, as far as the [declaration](https://github.com/kubescape/kubescape/blob/076aa7f8fe66e183f61b4023ea1f9b85751d0704/core/pkg/opaprocessor/utils.go#L100), is [alligned](https://github.com/kubescape/kubescape/blob/076aa7f8fe66e183f61b4023ea1f9b85751d0704/core/pkg/opaprocessor/processorhandler.go#L294) with the [definition](https://github.com/kubescape/kubescape/blob/076aa7f8fe66e183f61b4023ea1f9b85751d0704/core/pkg/opaprocessor/utils.go#L104) it should work

Reply:
yes it will not affect